Source/Dafny/Compilers/SinglePassCompiler.cs

```diff
@@ -3423,6 +3423,7 @@ public class CheckHasNoAssumes_Visitor : BottomUpVisitor {
         var b = (ComprehensionExpr.MapBoundedPool)bound;
         TrParenExpr(su.Substitute(b.Map), collectionWriter, inLetExprBody, wStmts);
         GetSpecialFieldInfo(SpecialField.ID.Keys, null, null, out var keyName, out _, out _);
+        // TODO: Shouldn't this be UniqueElements?
```

Comment:
Why should it be so? `Elements` are already unique, aren't they?

Reply:
Actually you're right, I confused this with the context of a map comprehension. Will remove.